tests/cxx/badinc.cc

```diff
@@ -1841,7 +1841,6 @@ int main() {
   // IWYU: I1_Class needs a declaration
   H_TemplateFunction<I1_Class*>(&i1_class);
   H_TemplateFunction(&i1_class);
-  // IWYU: I1_Enum is...*badinc-i1.h
```

Comment:
Cool that this disappears! Is it because the method is out-of-line-defined and the method decl thereby mentions the template argument?